include/g_local.h

```diff
@@ -1222,3 +1222,6 @@ extern qbool initial_match_spawns;
 extern int maxPlayerCount;
 
 #define AUTOTRACK_POWERUPS_PREDICT_TIME 2
+
+// Allow monster spawns in any mode, for example as details.
+#define KSPAWN_MONSTER(x) ((int) x->s.v.spawnflags == 1792)
```

Comment:
Would be nice to point out `256|512|1024 = (not easy | not normal | not hard) = 1792` somehow, I guess.

Reply:
Good point! I'll add some additional defines.

Reply:
Also, I think it should be (spawnflags & 1792) == 1792 since you probably could set some other flags on such entity? TBH quite cryptic way suggested, is it adopted by some other mod as well? Probably could be figured something other, like a new field.

Reply:
I'm poking the crmod people about it.